### ISSUE TITLE
Generic.WhiteSpace.ScopeIndent false positives when indenting open tags at a non tab-stop

### ISSUE THREAD

**Issue by cordis-dev** [squizlabs/PHP_CodeSniffer#2586]
PHPCS version:

`PHP_CodeSniffer version 3.4.2 (stable) by Squiz (http://www.squiz.net)`

Command:

`phpcbf --standard=PSR2 test.php -p`

File:
```php
     <?php if (true) : ?>
         <span></span>
        <?php endif ?>
```
Results:
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
test.php                                                 FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 70ms; Memory: 6MB
```

**Comment by gsherwood** [comment]
The Squiz.WhiteSpace.ScopeClosingBrace and Generic.WhiteSpace.ScopeIndent are conflicting here, due to the 5-space indent of the first line.

A quick workaround is to use a 4-space indent, which is what the sniffs are expecting when you don't specify `--tab-width`.

I'm not sure which sniff should be changed to resolve the conflict. It depends on if the opening PHP tag should be positioned at an exact tab stop or not.

**Comment by gsherwood** [comment]
Thanks for reporting this. I've committed a fix, which will be in 3.5.3. With this change, the open tags will try and align as normal, meaning that the code will be fixed like this:
```
     <?php if (true) : ?>
         <span></span>
     <?php endif ?>
```
